modules/gapi/include/opencv2/gapi/gcomputation.hpp

```diff
@@ -250,8 +250,8 @@ class GAPI_EXPORTS GComputation
     void apply(GRunArgs &&ins, GRunArgsP &&outs, GCompileArgs &&args = {});       // Arg-to-arg overload
 
     /// @private -- Exclude this function from OpenCV documentation
-    void apply(const std::vector<cv::gapi::own::Mat>& ins,                        // Compatibility overload
-               const std::vector<cv::gapi::own::Mat>& outs,
+    void apply(const std::vector<cv::Mat>& ins,                        // Compatibility overload
```

Comment:
Please keep the alignment for that "overload" comment

Reply:
Fixed.